lib/components/Upload.jsx

```diff
@@ -158,7 +158,7 @@ var Upload = React.createClass({
     return (
       <div className="Upload-button">
         <button
-          className="btn btn-secondary"
+          className="btn btn-primary"
```

Comment:
Can you include some screenshots of using the primary buttons for the uploads in the PR? My interpretation of my conversation with @HowardLook and @brandonarbiter was that they were looking for tweaks to the existing styles to bring out the state contrasts better, not switching to a different set, but it's possible that was just my interpretation.